problem1.py

```diff
@@ -1 +1,32 @@
 
+""""
+Name: TournamentTracker.py
+
+Purpose: This program will track the player wins and determine which group a player is placed in.
+
+Author: Nguyen.D 
+
+Created: 2021-03-04
+"""
+
+#get the player win and losses
+player = (input("Enter the wins and losses for your team: "))
```

Comment:
no need to prompt user for number of wins or losses.  As per requirements, it is fixed at 6